packages/cli/src/utils.ts

```diff
@@ -146,6 +146,27 @@ export function langToExt(lang: string) {
     javascript: '.js',
     js: '.js',
     python: '.py',
+    c: '.c',
+    'c++': '.cpp',
+    coffeescript: '.litcoffee',
+    css: '.css',
```

Comment:
I'm not sure we need css (and probably some other languages, like html) here.

Reply:
I've removed `css`, `html`, and `dm`. Let me know if I should remove anything else as well.